### PR TITLE
Pm vertex click

### PR DESCRIPTION
**Pull request by henrikmaa**
Added pm:vertexclick event.

Very useful for me, maybe useful for someone else too.

### CODE REVIEW

File: src/js/Edit/L.PM.Edit.Line.js [hunk 1]
```diff
@@ -259,7 +260,9 @@ Edit.Line = Edit.extend({
     // first, make this middlemarker a regular marker
     newM.off('movestart');
     newM.off('click');
-
+    newM.on('click', () => {
+      newM.on('click', this._onVertexClick, this);
+    })
```
Comment (Falke-Design) [inline src/js/Edit/L.PM.Edit.Line.js:265]: This makes no sense for me. MiddleMarkers are also created over `_createMarker` so it has already the click event for `_onVertexClick`. Also it don't understand why the marker have to clicked twice?

File: README.md [hunk 2]
```diff
@@ -354,7 +354,7 @@ The following events are available on a layer instance:
 | pm:disable         | `e`    | Fired when edit mode on a layer is disabled                                                          | `layer`, `shape`                                                                                                 |
 | pm:vertexadded     | `e`    | Fired when a vertex is added                                                                         | `layer`, `indexPath`, `latlng`, `marker`, `shape`                                                                |
 | pm:vertexremoved   | `e`    | Fired when a vertex is removed                                                                       | `layer`, `indexPath`, `marker`, `shape`                                                                          |
-| pm:markerdragstart | `e`    | Fired when dragging of a marker which corresponds to a vertex starts                                 | `layer`, `indexPath`, `markerEvent`, `shape`                                                                     |
```
Comment (Falke-Design) [inline README.md:357]: Why is this removed?

**Review by Falke-Design — CHANGES_REQUESTED** [review]

File: src/js/Edit/L.PM.Edit.Line.js (hunk 1 above)
Comment (henrikmaa) [reply src/js/Edit/L.PM.Edit.Line.js:265]: Thats what I tought aswell, but if I create a marker from middlemarker it does not have the event.

Comment (henrikmaa) [reply src/js/Edit/L.PM.Edit.Line.js]: And 2 click events was my bad. fixed

**Comment by Falke-Design** [comment]
Feature Request: #692
